### ISSUE TITLE
reset universe option not available

### ISSUE THREAD

**Issue by FerdinandoPucci** [fiji/3D_Viewer#29]
Current user guide does not match anymore the plugin.  Reset universe coordinate is not available in settings or anywhere in the menu anymore.  Overall, quite promising but also unusable as of now.

**Comment by tinevez** [comment]
Hello.

I am afraid this plugin is not maintained not supported anymore. 
Hopefully somebody will tell me wrong.

**Comment by ctrueden** [comment]
Yes, the author of this plugin has not maintained it for many years now. I suggest you take a look at [sciview](https://github.com/scenerygraphics/sciview/), which is where the current development effort is being invested. Now is a great time to make feature requests and bug reports to the sciview developers, as they are actively working on it.

**Comment by FerdinandoPucci** [comment]
Thank you!
